Mixmatch/streetview_dataset/parse_data_to_tfrecord_lib_test.ipynb

```diff
@@ -0,0 +1,239 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Parse_data_to_tfrecord_library Test\n",
+    "\n",
+    "This file consists several function test for the functions in the Parse_data_to_tfrecord_lib file."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from parse_data_to_tfrecord_lib import img_to_example, read_tfrecord, generate_tfexamples_from_detections, batch_read_write_tfrecords\n",
+    "from PIL import Image  # used to read images from directory\n",
+    "import tensorflow as tf\n",
+    "import os\n",
+    "import io\n",
+    "import IPython.display as display"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Test function img_to_example():\n",
+    "The function reads in image files and outputs tf.examples. The output should store the information feeded into the function."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "IMG_PATH = './TC11/svt1/img/19_00.jpg'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "try:\n",
+    "    img = Image.open(IMG_PATH, \"r\")\n",
+    "except Exception as e:\n",
+    "    print(e)\n",
+    "    print(IMG_PATH + \" is not valid\")\n",
+    "\n",
+    "example = img_to_example(img, label=0)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Testing\n",
+    "# The label feature should be a int64_list with value of {value: 0}\n",
+    "print(example.features.feature['label'])\n",
```

Comment:
I'd prefer if you use a `test_passed` boolean and if-conditions to explicitly indicate what is expect if possible. e.g.
```
test_passed = true
if len(example.features.feature['label']) != 1:
  test_passed = false

for label in example.features.feature['label']:
  if label != 0:
    test_passed = false

if test_passed:
  print("Test passed!")
```

Reply:
Done. Added a decoder to the tf.example, and assert the results.